lib/main_publiccloud.pm

```diff
@@ -49,10 +49,13 @@ sub load_maintenance_publiccloud_tests {
         if (get_var('PUBLIC_CLOUD_CONSOLE_TESTS')) {
             load_extra_tests_prepare();
             load_publiccloud_consoletests();
-        }
-        if (get_var('PUBLIC_CLOUD_CONTAINERS')) {
+        } elsif (get_var('PUBLIC_CLOUD_CONTAINERS')) {
             load_podman_tests() if is_sle('>=15-sp1');
             load_docker_tests();
+        } elsif (get_var('PUBLIC_CLOUD_XFS')) {
```

Comment:
maybe we should add same section to `latest` too ?

Reply:
Good point! I've added it there as well.